.github/workflows/lint.yml

```diff
@@ -46,10 +46,11 @@ jobs:
         lint:
           - name: "ruff"
             commands: |
-              tox -e ruff -- check
+              tox -e ruff
           - name: "pylint"
             commands: |
               echo "::add-matcher::.github/workflows/matchers/pylint.json"
+              tox -e fastlint
```

Comment:
Is there a reason to run both fastlint and lint? It seems that fastlint is a subset of lint?

Reply:
It behaves slightly differently than regular lint and has been broken two times in the last few weeks. Pylint cannot figure out on its own that `instructlab.eval` is a 3rd party package, so we need to teach it every external instructlab package.